packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -288,15 +288,28 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   );
 
   /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
-  const onImageDownloadButtonClicked = useCallback((imageUrl: string, saveAsName: string): void => {
-    // Create a new anchor element
-    const a = document.createElement('a');
-    // Set the href and download attributes for the anchor element
-    a.href = imageUrl;
-    a.download = saveAsName || 'download';
```

Comment:
Would only add `a target='_blank'` attribute to the anchor (<a>) tag work here?
```
const a = document.createElement('a');
    
    a.href = imageUrl;

    a.target = '_blank';  // This instructs the browser to open the link in a new tab/window.

    a.download = saveAsName || 'download';
    
    document.body.appendChild(a);
    
    a.click();
    
    a.remove();
```

Reply:
No on iOS browsers this does not work. The prompt is controlled by the native iOS, and because it sees that file type is a `png` it will always give the `View` option. So we try to open it as a pop up instead using the `FileReader` This will have another prompt instead, asking users if they want to `Allow` or `Block`. If they hit `Allow` they are navigated to a new page for just the image where users will have the opportunity to save it themselves.

Reply:
Oh...I see. That's tricky. 
 I briefly checked Outlook's mobile web to see their approach. It seems they utilize a `Route` to create a new tab with a download URL, which is used for both previewing and downloading the content. But we don't have `Route` set up inside the component.